From cd3aad45d6c71341bc176bafcacf8e83ad9f454e Mon Sep 17 00:00:00 2001
From: Li Jun <jun.li@nxp.com>
Date: Tue, 8 Aug 2017 21:34:19 +0800
Subject: [PATCH 2428/5242] MLK-16013-41 ARM64: imx8mq: remove
 dis_u2_susphy_quirk for usb

commit  f5bf7e45e9cb8dfea87b9e0842802012f3356779 from
https://source.codeaurora.org/external/imx/linux-imx.git

As the USB3 hotplug issue identified by IC team, the recommended
solution is to use FREECLK by clear COMMMONONN, so remove this
quick temp workaround.

Reviewed-by: Peter Chen <peter.chen@nxp.com>
Signed-off-by: Li Jun <jun.li@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi |    2 --
 1 file changed, 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
index a846610..fdbdf7a 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
@@ -667,7 +667,6 @@
 			phy-names = "usb2-phy", "usb3-phy";
 			power-domains = <&usb_otg1_pd>;
 			snps,power-down-scale = <2>;
-			snps,dis_u2_susphy_quirk;
 			status = "disabled";
 		};
 	};
@@ -707,7 +706,6 @@
 			phy-names = "usb2-phy", "usb3-phy";
 			power-domains = <&usb_otg2_pd>;
 			snps,power-down-scale = <2>;
-			snps,dis_u2_susphy_quirk;
 			status = "disabled";
 		};
 	};
-- 
1.7.9.5

